src/components/Card/AddressHashCard.tsx

```diff
@@ -109,6 +122,7 @@ export default ({ title, hash, dispatch }: { title: string; hash: string; dispat
       >
         <img src={CopyIcon} alt="copy" />
       </div>
+      <div id="address_hash__hash">{hash}</div>
```

Comment:
hash__hash, a bit weird name~

Reply:
I have modified the name. https://github.com/nervosnetwork/ckb-explorer-frontend/pull/168/commits/6ba488ca7dc7217a70d48c14a508b78f4bc9080a